clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/jobs/local/StreamConsumer.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.jobs.local;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public interface StreamConsumer {
```

Comment:
Who is responsible for closing an InputStream passed to an implementation of this interface? The implementation or the caller? Can documentation on that contract be added?

Reply:
Pushed this responsibility to the implementing class and added this to the class documentation.